### DISCUSSION TITLE
API Versioning

### DISCUSSION THREAD

**Discussion by AndrewMarkUK** [dedoc/scramble#182] [Q&A] [answered]
Firstly, absolutely love this! Amazing and really easy to implement. So refreshing to find developer/s that have not ploughed in 'over-complexity' in to their product. Well done!

I have question as I am putting together a strategy for versioning my API. Firstly, for the controller structure In Laravel I am doing:

Controllers/API/v1/ProductController.php
Controllers/API/v2/ProductController.php

In the api.php (route), I am doing this:

use App\Http\Controllers\API\v1\ProductController as V1ProductController;
use App\Http\Controllers\API\v2\ProductController as V2ProductController;

// Version 1
Route::namespace('API\v1')->group(function () {
    Route::resource('v1/products', V1ProductController::class);
});

// Version 2
Route::namespace('API\v2')->group(function () {
    Route::resource('v2/products', V2ProductController::class);
});

Now, when I visits the docs/api, I can see all my api resource methods links in the left column as expected, something like:

Product -
v1.product.index
v1.product.store
v2.product.index
v2.product.store

Is there an implementation for placing the v1 and v2 in their own drop down so it looks something ;

Product -
v1 -
v1.product.index
v1.product.store
v2 -
v2.product.index
v2.product.store

Or, is it a case of somehow having separate api/docs for each version AND if so, how is that done.

Any hints, tips on this is much appreciated

Thank you.

**Comment by romalytvynenko** [comment, marked as answer]
@AndrewMarkUK Released in `v0.10.*`. https://scramble.dedoc.co/usage/multiple-docs
